packages/cli/src/databases/dsl/index.ts

```diff
@@ -0,0 +1,17 @@
+import type { QueryRunner } from 'typeorm';
+import { Column } from './Column';
+import { CreateTable, DropTable } from './Table';
+import { CreateIndex, DropIndex } from './Indices';
+
+export const createSchemaBuilder = (tablePrefix: string, queryRunner: QueryRunner) => ({
```

Comment:
Should we test this?

Reply:
This whole thing definitely needs better unit and integration tests. But, I'd like to do that in a separate PR, when I port other the other helper methods like `addColumns`, `dropColumns`, `fetchIds`, and `insertInto`.